test/performance/range/views/view_minimiser_hash_benchmark.cpp

```diff
@@ -0,0 +1,118 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/range/views/minimiser_hash.hpp>
+#include <seqan3/test/performance/sequence_generator.hpp>
+#include <seqan3/test/performance/naive_minimiser_hash.hpp>
+#include <seqan3/test/performance/units.hpp>
+
+inline benchmark::Counter bp_per_second(size_t const basepairs)
+{
+    return benchmark::Counter(basepairs,
+                              benchmark::Counter::kIsIterationInvariantRate,
+                              benchmark::Counter::OneK::kIs1000);
+}
+
+inline seqan3::shape make_gapped_shape(size_t const k)
+{
+    seqan3::shape shape_{};
```

Comment:
Why do we use an underscore at the end of the variable name?

Reply:
I do not know, I have copied that from the kmer_hash_benchmark